src/combinatorics/simple_graph/prod.lean

```diff
@@ -20,6 +20,10 @@ two edges is a square.
 ## Notation
 
 * `G □ H`: The box product of `G` and `H`.
+
+## TODO
+
+Define all other graph products!
```

Comment:
Which other ones? (the comment should elaborate on that)

Reply:
Oh I realize this isn't actually something you changed in this PR, but the comment stands.